app/src/main/java/org/andsopt/android/yes24ticket/presentation/ui/component/InformationFooter.kt

```diff
@@ -0,0 +1,237 @@
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.andsopt.android.yes24ticket.R
+import org.andsopt.android.yes24ticket.presentation.ui.component.InformationFooterConstants.END_INDEX
+import org.andsopt.android.yes24ticket.presentation.ui.component.InformationFooterConstants.START_INDEX
+import org.andsopt.android.yes24ticket.ui.theme.YES24TICKETTheme
+import org.andsopt.android.yes24ticket.ui.theme.defaultYes24TicketTypography
+
+@Composable
+fun InformationFooter(
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier =
+            modifier
+                .fillMaxWidth()
+                .padding(top = 10.dp, bottom = 19.dp),
+    ) {
+        FooterCSAndPCButton()
+
+        DividerBetweenText(
+            startText = R.string.information_footer_privacy_policy,
+            endText = R.string.information_footer_terms,
+            padding = 8,
+            textColor = Color.Black,
+            textStyle = defaultYes24TicketTypography.bodyRegular12,
+        )
+
+        FooterCompanyInformation()
+
+        Spacer(Modifier.height(2.dp))
+
+        DividerBetweenText(
+            startText = R.string.information_footer_business,
+            endText = R.string.information_footer_report,
+        )
+
+        Spacer(Modifier.height(2.dp))
+
+        DividerBetweenText(
+            startText = R.string.information_footer_email,
+            endText = R.string.information_footer_service_provider,
+        )
+
+        Spacer(Modifier.height(2.dp))
+
+        Text(
+            text = stringResource(R.string.information_footer_copyright),
+            style = defaultYes24TicketTypography.captionSemiBold10,
+            color = Color.Gray,
+            modifier = Modifier.align(Alignment.CenterHorizontally),
+        )
+
+        FooterLinkButton()
+    }
+}
+
+@Composable
+fun DividerBetweenText(
+    @StringRes startText: Int,
+    @StringRes endText: Int,
+    modifier: Modifier = Modifier,
+    padding: Int = 6,
+    textColor: Color = Color.Gray,
+    textStyle: TextStyle = defaultYes24TicketTypography.captionSemiBold10,
+) {
+    Row(
+        horizontalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.CenterVertically,
+        modifier =
+            modifier
+                .fillMaxWidth(),
+    ) {
+        Text(
+            text = stringResource(startText),
+            style = textStyle,
+            color = textColor,
+            modifier = Modifier.padding(end = padding.dp),
+        )
+
+        HorizontalDivider(
+            modifier =
+                Modifier
+                    .width(1.dp)
+                    .height(9.dp)
+                    .background(Color.Red),
+        )
```

Comment:
```suggestion
        VerticalDivider(
            modifier =
                Modifier
                    .height(9.dp)
                    .background(Color.Red),
                thickness = 1.dp,
                color = Color.Red
        )
```

`VerticalDivider`를 쓰는 것이 더 용도가 명확할 거 같습니다...!!

Reply:
color를 2번 선언하는 이유가 있나요?

Reply:
와 제가 원래 VerticalDivider를 썼다가 눈물을 머금고 HorizontalDivider로 바꾼 이유가 있거든요 color가 안 먹거나 padding이 안 먹거나 해서!!!!! 근데 width가 아니라 thickness를 쓰면 되는 거였구나!!! 원래 제 코드 알려드리겠어요

```agsl
VerticalDivider(
            modifier =
                Modifier
                    .background(Color.Red) // 위에 선언하든 height 아래에 선언하든 안됨ㅋㅋ
                    .width(1.dp)
                    .height(9.dp)
                    
        )
```

Divider쓸 때 width height 주의하세요ㅋㅋ ㅠㅠ
좋은 거 배워갑니다 상일님,,

Reply:
@beom84 컬러 두번 선언한 건 실수이신듯용!!